src/explore-education-statistics-admin/src/pages/admin-dashboard/AdminDashboardPage.tsx

```diff
@@ -153,14 +155,15 @@ const AdminDashboardPage = () => {
               >
                 {release => (
                   <>
-                    <SummaryListItem
-                      key="select"
-                      term="Select pre release access"
+                    <FormFieldset
```

Comment:
Is this output OK?  I think it'll be outputting a Fieldset and a SummaryList within another SummaryList.  Is this valid?

Reply:
It should sit outside the summary list, can show you the resulting html locally